app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    file_name = command.split()
+    if file_name[0] == "mv":
+        if "/" in file_name[2]:
+            os.makedirs(file_name[2])
+        else:
```

Comment:
you need to move file with all fufilness now you create file you need but original file is still exists and created file is empty, or I missing something?